ui/actions/nuxeo-delete-document-button.js

```diff
@@ -51,6 +51,14 @@ import '../nuxeo-button-styles.js';
       return html`
         <style include="nuxeo-action-button-styles nuxeo-button-styles"></style>
 
+        <nuxeo-document
+          id="doc"
+          doc-id="[[parentDoc.uid]]"
+          enrichers="permissions"
+          auto
+          response="{{_enrichedParent}}"
+        ></nuxeo-document>
```

Comment:
Since this introduces a new request, I'd recommend moving away from this solution, as it will introduce a performance penalty and visual quirks (i.e., delays in making the action visible or not).